__main__.py

```diff
@@ -33,5 +33,32 @@
 
 from libiocage.cli import cli
 
+
+def main_safe():
+  try:
+    main()
+  except BaseException as e:
+    return e
+
+
+def main():
+  cli(prog_name="iocage")
+
+
 if __name__ == "__main__":
-    cli(prog_name="iocage")
+
+  debug = False
+
+  try:
+    coverdir = os.environ["IOCAGE_TRACE"]
+    import trace
```

Comment:
This should be imported at the top.

Reply:
We only need it for debugging, so I want to only include it when it's required.